watertap/unit_models/tests/test_pressure_changer.py

```diff
@@ -354,6 +402,7 @@ def test_solution(self, Pump_frame):
         assert pytest.approx(1, rel=1e-3) == value(m.fs.unit.eta_ratio[0])
 
         # Test low bep flow case
+        m.fs.unit.flow_ratio.unfix()
         m.fs.unit.bep_flow.fix(default_flow_vol / 2)
         results = solver.solve(m)
```

Comment:
This seems to be duplicating the solve that is now done in an earlier test. Ideally, this test would just check the values in the model based on the previous solve.

Reply:
For the default case (line 402), the initial solve is used. The additional solves test different regions of the step-wise flow-efficiency correlation.

Reply:
@andrewlee94 - We used a nested `Expr_if `to account for 3 different regions of a curve representative of pump efficiency ratio as a function of pump flow ratio and wanted to ensure each region worked as expected.